tiling.fnl

```diff
@@ -0,0 +1,126 @@
+(local hhtwm (require :hhtwm))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Swap Windows
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(fn swap-window
+  [arrow]
+  "
+  Swap window using hhtwm.
+  "
+  (let [dir {:h "west" :j "south" :k "north" :l "east"}
```

Comment:
Why encode the hjkl part here? e.g. Why not just have `swap-window-left` call `swap-window :left`?

Reply:
I mean `swap-window :west`. You're adding an intermediate hjkl to compass mapping, when you could encode a 'left right up down' to compass mapping directly in the function calls.